.github/workflows/ci.yml

```diff
@@ -16,3 +16,5 @@ jobs:
     uses: voxpupuli/gha-puppet/.github/workflows/beaker.yml@v1
     with:
       pidfile_workaround: 'false'
+      unit_runs_on: 'cern-self-hosted'
+      pidfile_workaround: 'cern-self-hosted'
```

Comment:
This line is surprising. What happened here?

Reply:
had a problem with the template. fixed it in https://github.com/voxpupuli/modulesync_config/pull/840